src/TulsiGenerator/PBXTargetGenerator.swift

```diff
@@ -108,6 +108,14 @@ extension PBXTargetGeneratorProtocol {
   }
 }
 
+/// Assume external paths in Bazel are exported as ".."
+private func patchLeadingDotsForExternal(_ value: String) -> String {
+  if value.hasPrefix("..") {
```

Comment:
This is necessary anymore given that the latest version of Bazel is no longer using dots, and already exports these files as `external`.